.travis.yml

```diff
@@ -12,7 +12,9 @@ install:
   - npm install
 before_script: greenkeeper-lockfile-update
 script:
+  - set -e
```

Comment:
this isn't required anymore is it?

Reply:
that is for travis not stopping the build as soon as it gets a non 0 exit code, I don't want the build to go through if one of the scripts fail, I'll try remove it and see if the problem is gone now

Reply:
After testing with the set -e if the node scripts exits with 1 it'll stop the build there, otherwise itll go through to the gradle bit and the CI will fail, fail fast it's much better in this case